src/rust/ide/src/double_representation/definition.rs

```diff
@@ -109,6 +75,11 @@ pub fn locate(ast:&ast::known::Module, id:&Id) -> FallibleResult<ChildDefinition
 at least one non-empty line.")]
 struct MissingLineWithAst;
 
+#[allow(missing_docs)]
+#[derive(Fail,Clone,Debug)]
+#[fail(display="Cannot find definition child by id {:?}.",_0)]
```

Comment:
I believe the sentence should be "Cannot find definition child WITH id ..."

Reply:
changed to with